custom_components/birthdays/__init__.py

```diff
@@ -13,15 +13,20 @@
 
 _LOGGER = logging.getLogger(__name__)
 
-CONF_NAME = 'name'
-CONF_DATE_OF_BIRTH = 'date_of_birth'
-CONF_ICON = 'icon'
+CONF_UNIQUE_ID            = 'unique_id'
+CONF_NAME                 = 'name'
```

Comment:
Same here with the spaces like lines 70-73, could there be some autoformatter messing with the spaces? :)

Reply:
Yes, you are correct about that one, I had some automatic alignment thingy in VSCode. :-) it's good enough for formatting but could cause trouble